en/computer_vision/visual_inertial_odometry.md

```diff
@@ -0,0 +1,56 @@
+# Visual Intertial Odometry (VIO)
+
+*Visual Inertial Odometry* (VIO) is a [computer vision](../computer_vision/README.md) technique used for estimating the 3D *pose* (local position and orientation) and *velocity* of a moving vehicle relative to a *local* starting position.
+It is commonly used to navigate a vehicle in situations where GPS is absent or unreliable (e.g. indoors, or when flying under a bridge).
+
+VIO uses [Visual Odometry](https://en.wikipedia.org/wiki/Visual_odometry) to estimate vehicle *pose* from camera images, combined with inertial measurements from the vehicle IMU (to correct for errors associated with rapid vehicle movement resulting in poor image capture).
+
+<!-- video here -->
+
+
+
+## Supported Sensors 
+
+- [T265 Intel Realsense Tracking Camera](../peripherals/camera_t265_vio.md)
+
+
+## Setup Instructions
+
+You will need to attach your camera to the companion computer and set up ROS.
+Generally the camera should be mounted facing down. 
+If using the [T265 ](../peripherals/camera_t265_vio.md) it should also be soft-mounted.
+
+
+> **Note** ROS is not required to for VIO, but is the supported method. 
+
+These instructions explain how to set up VIO using ROS:
+- On the companion computer, install and configure [MAVROS](https://dev.px4.io/master/en/ros/mavros_installation.html) and verify it is connected successfully to the flight controller.
```

Comment:
@baumanta That link does not explain how to verify if MAVROS is successfully connected. How do you do that?

Reply:
So after hearing that I would go to the mavlink console on QGC and check for the HEARTBEAT. Mavros/Offboard-component has a unique SYSTEM_ID and you could check if you get its heartbeat. Not sure if that's what the author meant, though.

Reply:
@baumanta Thank you. The SYSTEM_ID isn't a good thing for instructions because it changes based on setup. Can you confirm what ROS/our setup use for the MAV_TYPE or component ID, as these would be much easier to look at?

For example, if the [MAV_COMP_ID_VISUAL_INERTIAL_ODOMETRY](https://mavlink.io/en/messages/common.html#MAV_COMP_ID_VISUAL_INERTIAL_ODOMETRY) is used, that would be eaiser to check.

Reply:
PS, there are instructions in a different doc, but I think you could only use these when you have got avoidance running: https://dev.px4.io/master/en/ros/external_position_estimation.html#check-external-estimate

So again, would really appreciate your advice.

Reply:
yes I would agree there to move the "check" to the end of the instructions. After the nodes are launched. It is confusing where it is because you are right at that point nothing is running yet. Mavlink Inspector is a good advice there you could basically check for any topis unique to VIO: The VIO heard beat (197=MAV_COMP_ID_VISUAL_INERTIAL_ODOMETRY), or the VIO data topic itsself (not sure how it is called but there is  mavlink message with the odometry data going to PX4